.circleci/config.yml

```diff
@@ -532,57 +536,77 @@ workflows:
           filters:
             tags: &filters-release-tags
               only: /^[0-9]+\.[0-9]+\.[0-9]+(-[a-zA-Z0-9]+)?/
+          context:
+            - dockerhub-quorumengineering-ro              
       - spotless:
           filters:
             tags:
               <<: *filters-release-tags
+          context:
+            - dockerhub-quorumengineering-ro              
       - windowsBuild:
           filters:
             tags:
               <<: *filters-release-tags
+          context:
+            - dockerhub-quorumengineering-ro              
       - referenceTestsPrep:
           requires:
             - assemble
           filters:
             tags:
               <<: *filters-release-tags
+          context:
+            - dockerhub-quorumengineering-ro              
       - referenceTests:
           requires:
             - assemble
             - referenceTestsPrep
           filters:
             tags:
               <<: *filters-release-tags
+          context:
+            - dockerhub-quorumengineering-ro              
       - unitTests:
           requires:
             - assemble
           filters:
             tags:
               <<: *filters-release-tags
+          context:
+            - dockerhub-quorumengineering-ro              
       - propertyTests:
           requires:
             - assemble
           filters:
             tags:
               <<: *filters-release-tags
+          context:
+            - dockerhub-quorumengineering-ro              
       - integrationTests:
           requires:
             - assemble
           filters:
             tags:
               <<: *filters-release-tags
+          context:
+            - dockerhub-quorumengineering-ro              
       - acceptanceTests:
           requires:
             - assemble
           filters:
             tags:
               <<: *filters-release-tags
+          context:
+            - dockerhub-quorumengineering-ro              
```

Comment:
Is this context actually available to PRs from forks? Previously doing it this way has broken the ability for CI to build PRs.

Reply:
Quite right, I'll fix this up in w3s and just add a comment here re the above